docs/addons/keda.md

```diff
@@ -0,0 +1,146 @@
+# Kubevious Add-on
+
+This add-on installs [Keda](https://github.com/kedacore/keda) Kubernetes-based Event Driven Autoscaling.
+
+KEDA allows for fine-grained autoscaling (including to/from zero) for event driven Kubernetes workloads. KEDA serves as a Kubernetes Metrics Server and allows users to define autoscaling rules using a dedicated Kubernetes custom resource definition.
+
+## Usage
+
+```typescript
+
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+# Normal Usage
+# const addOn = new blueprints.addons.KedaAddOn();
+# In case of AWS SQS, there is a workaround required when initializing keda (Please refer https://github.com/kedacore/keda/issues/837#issuecomment-789037326 )
+const kedaParams = {
+    podSecurityContextFsGroup: 1001,
+    securityContextRunAsGroup: 1001,
+    securityContextRunAsUser: 1001,
+    enableIRSA: true,
+    irsaRoles: ["CloudWatchFullAccess", "AmazonSQSFullAccess"]
+}
+const addOn = new blueprints.addons.KedaAddOn(kedaParams)
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `version`: Version fo the Helm Chart to be used to install Kubevious
+- `kedaOperatorName`: Name of the KEDA operator
+- `enableIRSA`: Specifies whether a service account should be created by by CDK with IRSA. If provided false, Service Account will be created by Keda without IRSA
+- `createServiceAccount`: Specifies whether a service account should be created by Keda. 
+- `kedaServiceAccountName`: The name of the service account to use. If not set and create is true, a name is generated.
+- `podSecurityContextFsGroup`: PodSecurityContext holds pod-level security attributes and common container settings. fsGroup is a special supplemental group that applies to all containers in a pod. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+- `securityContextRunAsUser`: SecurityContext holds security configuration that will be applied to a container. runAsUser is the UID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+- `securityContextRunAsGroup`: SecurityContext holds security configuration that will be applied to a container. rusAsGroup The GID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+
+- `values`: Arbitrary values to pass to the chart. Refer to the Kubevious [Helm Chart documentation](https://github.com/kubevious/helm) for additional details
```

Comment:
Please replace reference to Kubevious to Keda repo.

Reply:
fixed